addons/website_slides/models/slide_slide.py

```diff
@@ -171,7 +171,10 @@ class Slide(models.Model):
     # google
     google_drive_id = fields.Char('Google Drive ID of the external URL', compute='_compute_google_drive_id')
     # content - webpage
-    html_content = fields.Html("HTML Content", help="Custom HTML content for slides of category 'Article'.", translate=True, sanitize_attributes=False, sanitize_form=False)
+    html_content = fields.Html(
+        "HTML Content", translate=True,
+        sanitize_attributes=False, sanitize_form=False, sanitize_overridable=True,
```

Comment:
We are adding in stable an attribute to a field which indeed should have been set in the PR that you linked.
What I am afraid is that adding such an attribute could break client DBs on this field.
Have you tried a migration to test if anything breaks ?

Reply:
Hello @thjo-odoo 

Since the attribute here define how the HTML field will be sanitized, the already sanitized fields will not be impacted at all here. :smiley:

Reply:
If you are sure there will be no impact then we are good to go.